lib/ansible/modules/cloud/azure/_azure.py

```diff
@@ -51,7 +51,6 @@
     description:
       - azure role size for the new virtual machine (e.g., Small, ExtraLarge, A6). You have to pay attention to the fact that instances of
         type G and DS are not available in all regions (locations). Make sure if you selected the size and type of instance available in your chosen location.
-    default: Small
```

Comment:
why remove the default value

Reply:
because it's not defined in parameters specification, and that's why sanity test failed